hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/keys/CopyKeyHandler.java

```diff
@@ -89,11 +92,17 @@ protected void execute(OzoneClient client, OzoneAddress address)
               OZONE_REPLICATION_TYPE_DEFAULT));
     }
 
+    Map<String, String> keyMetadata = new HashMap<>();
+    String gdprEnabled = bucket.getMetadata().get(OzoneConsts.GDPR_FLAG);
+    if (Boolean.parseBoolean(gdprEnabled)) {
+      keyMetadata.put(OzoneConsts.GDPR_FLAG, Boolean.TRUE.toString());
+    }
```

Comment:
One more thing: I think instead of building key metadata from scratch, `cp` should copy actual metadata of the existing key, which it can get via `OzoneKeyDetails` (except `GDPR_SECRET` and `GDPR_ALGORITHM` should be removed).  This would ensure that any user-supplied custom metadata is transferred, as well as reduce the need for future changes in case new default metadata is added in `PutKeyHandler`.

Reply:
Thank you for your advice, done.